lib/user/jwt/client.js

```diff
@@ -316,17 +309,13 @@ class Client {
    *
    **/
   createRequestOptions (urlPattern, context, data = {}, isGET, options = {}) {
-    const accessToken = this.generateAccessToken(data, this.serviceToken, 'HS256')
     const accessTokenV2 = this.generateAccessToken(data, this.privateKey(), 'RS256', options)
     const url = this.createEndpointUrl(urlPattern, context)
     const hasData = !!Object.keys(data).length
 
     const requestOptions = {
       url,
-      headers: {
-        'x-access-token': accessToken,
-        'x-access-token-v2': accessTokenV2
-      },
+      headers: { 'x-access-token-v2': accessTokenV2 },
```

Comment:
Just want to double-check that `{ 'x-access-token-v2': accessTokenV2 }` is correct (I thought I saw a change from `x-access-token-v2` to `x-access-token` in some code earlier, not sure if I did or if the context is _entirely_ different)

Reply:
(Otherwise lovely, ta)

Reply:
So we decided to hang onto the `-v2` just to make it easier to identify. Just double checked the previous PR's and we've just removed the `x-access-token` so far 🤞